server/lib/nl/explore/params.py

```diff
@@ -49,6 +49,7 @@ class DCNames(str, Enum):
   SDG_DC = 'sdg'
   SDG_MINI_DC = 'sdgmini'
   UNDATA_DC = 'undata'
+  UNDATA_ILO_DC = 'undata_ilo'
```

Comment:
We should also add this to `SPECIAL_DC_LIST`

Reply:
Added